[week2] 블로그 작성 (yeji)

YJ9985 opened the pull request.

YJ9985 committed:
Create yeji_link.md

kyungseopk1m commented:
말씀해주신 공룡책 목차를 쭉 읽어봤는데 구성이 정말 탄탄하네요.
830페이지라니.. 흡사 벽돌책이라고 불러도 되겠어요.
저는 프로세스 동기화 단락의 `Mutex Lock` 개념이 제일 궁금하네요. 한번쯤은 깊게 파보고 싶은 개념이라 생각했었거든요.
고랭에서도 뮤텍스가 있는데, 사용하면서 `동시성 제어를 통해 일관성을 유지한다`는 맥락에서 DB에서의 트랜잭션 개념과 조금은 유사하지 않나 싶었어요.
물론 목적성 자체가 완전히 다르지만요. 다 읽고 체화하면 확실히 도움이 많이 되겠네요.
글 올려주시면 그때 다시 피드백하도록 하겠습니다!

YJ9985 committed:
Update yeji_link.md

미루고 미루다 드디어 작성 완료한 블로그 글입니다..ㅎ

운영체제 부수기를 메인 주제로 꾸준히 작성할 계획이고,
첫 번째 주제로 운영체제의 개념과 구조 쉽게 이해하기를 설정했습니다!

DevRunner21 commented:
안녕하세요!
대학교때 OS 강의 전공서적이 공룡책이었는데... 반갑네요 ㅋㅋ

혹시 강의 같은거 없이 책만 읽으시면서 공부하시는 건가요??
그 책이 내용은 좋은데 번역이 별로라서 그냥 읽으면 이해가 어려운 편이라고 기억하는데..

혹시 서브 자료 필요하시면 아래 강의 추천드립니다.
이화여대 교수님의 OS 강의인데 몇년전까지는 OS 공부하기 좋은 강의로 유명했었는데..
요즘은 자료가 많아져서 더 좋은게 있을지도 모르겠네요 ㅋㅋ

http://www.kocw.net/home/search/kemView.do?kemId=1046323

kyungseopk1m commented:
안녕하세요 예지님
올려주신 글 잘 읽었어요! 🙂

먼저 글을 읽으며 `운영체제에 대한 정의`, `컴퓨터의 정보 처리 방식`, `컴퓨터의 역사`, `프로그램에 대한 정의`에 대한 개요를 간단히 살펴볼 수 있었어요.
이러한 개념들은 운영체제 수업 외에는 접할 일이 거의 없었는데, 다시 한번 상기하고 이해하게 되어 좋았습니다.

특히 개념에 대한 정의를 내릴 때 원문을 먼저 써내려가고, 이후 한글로 설명하는 방식으로 진행하셨는데 저는 이 부분이 인상적이었어요.
개발을 하는 과정에 영미권의 레퍼런스나 공식문서를 볼 일이 많은데 원문이 함께 있으니 보다 개념에 대해 접근하기 수월했어요.

또한 단락별로 개념을 설명하면서 점차 깊이를 더해가는 방식으로 작성하셨는데, 예지님이 이러한 방식으로 설명해주신 덕분에 더욱 몰입해서 읽을 수 있었습니다.
확실히 이해하기도 더 쉬웠구요!

다소 아쉬웠던 부분은 개념에 대한 나열이 주로 이루어지다보니 독자 입장에서 글이 조금은 딱딱하게 느껴질 수 있겠다는 것이었어요.
예를 들어 `우리가 레시피를 보며 요리하듯이`와 같은 예시를 통해 독자의 이해에 도움을 주는 비유라든지, `폰 노이만 아키텍처`와 같은 이미지를 좀 더 사용한다면 독자가 개념을 이해하기 더욱 쉬워질 것 같아요.
저도 비슷한 글을 작성하면서 느꼈는데, 원론적인 개념을 이해하기 쉽게 풀어낸다는 건 참 어려운 문제 같아요.

책의 내용에 추상적인 개념도 많고, 원론적으로만 접근하는 부분도 많아서 예지님의 시각과 언어로 부가 설명을 덧붙인다면 더욱 좋을 것 같아요!
좋은 글 감사합니다. 잘 읽었습니다 :-)


여담으로 블로그 링크 올리실 때 단축 URL을 사용하셨는데, 티스토리에서 URL 설정을 바꿀 수 있어요!
티스토리 관리자 페이지에 들어가셔서 `관리 > 블로그 > 주소` 설정으로 가면 포스트 주소를 변경할 수 있답니다.
포스트 주소를 숫자로 설정하시면 url shortener 없이도 편리하게 올리실 수 있을 것 같아요ㅎㅎ
